src/runtime/cuda.cpp

```diff
@@ -676,9 +678,9 @@ WEAK int halide_cuda_device_free(void *user_context, halide_buffer_t *buf) {
             // We don't want to use a buffer freed one stream on
             // another, as there are no synchronization guarantees and
             // everything is async.
-            int result = halide_cuda_get_stream(user_context, ctx.context, &item->stream);
```

Comment:
This is a minor change in contract. If you override halide_cuda_get_stream you should now call halide_error if there's an error (or signal your error some other way). We should clarify the comment in HalideRuntimeCuda.h

Reply:
Done (also applied to opencl.cpp)